ui/pages/confirmations/components/confirm/info/__snapshots__/info.test.tsx.snap

```diff
@@ -81,7 +81,7 @@ exports[`Info renders info section for typed sign request 1`] = `
     class="mm-box mm-box--margin-bottom-4 mm-box--padding-0 mm-box--background-color-background-default mm-box--rounded-md"
   >
     <div
-      class="mm-box mm-box--padding-2"
+      class="mm-box"
```

Comment:
from the image and the update, it seems we moved padding from all sides which causes the text to be not aligned left and right to the text in the info section below. We should preserve the left and right padding

Reply:
Good catch @digiwand

Reply:
Good catch @digiwand , I updated the PR